cmd/podman/untag.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	untagCommand cliconfig.UntagValues
+
+	_untagCommand = &cobra.Command{
+		Use:   "untag [flags] IMAGE TAG [TAG...]",
```

Comment:
The syntax seems wrong. `untag [flags] IMAGE [NAME:[TAG]]...`?

Reply:
Hm, if we say that we reference a `name` to the full name including the tag, it would be
`untag [flags] IMAGE [NAME...]`, right?